features/afsluiten.feature

```diff
@@ -0,0 +1,73 @@
+#language: nl
+
+Functionaliteit: Zaakdossiers moeten na afsluiten duurzaam toegankelijk gehouden worden. Om dit te mogelijk te maken moet aan een aantal voorwaarden worden voldaan.
+  Als een informatiebeheerder
+  wil ik dat zaakdossiers voor het afsluiten van de zaak beschikken over een minimale set kenmerken
+  zodat ik deze dossiers goed kan beheren en eventueel op het juiste moment kan vernietigen
+
+Scenario: afsluiten zaak - zaak heeft geen resultaat
+  Gegeven dat een medewerker de zaak met uuid "095be615-a8ad-4c33-8e9c-c7612fbf6c9f" wil afsluiten door het aanmaken van de eindstatus.
+  Als de zaak geen resultaat (Zaak.Resultaat) heeft,
+  dan kan de zaak (nog) niet worden afgesloten - daarvoor moet door de consumer eerst een resultaat worden aangemaakt.
```

Comment:
vanuit een technisch perspectief bestaat (nog) niet niet. Je kan een zaak afsluiten of niet. Ik zou ook verwachten dat wat je hier achter de - zet in de scenario omschrijving komt te staan. 

Een zaak niet af kunnen sluiten is vermoedelijk ook een 400 status code? 

Je zou dan iets krijgen als:

Scenario: een zaak kan niet afgesloten voordat een consumer een resultaat heeft aangemaakt